packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,297 @@
+  withdrawals: BigDecimal!
+  "Total number of withdrawals made"
+  withdrawalCount: BigInt!
+  "Total rewards collected from farm"
+  harvest: BigDecimal!
+  "Total rewards collected from farm in USD"
+  harvestUSD: BigDecimal!
+  "Total number of harvests made"
+  harvestCount: BigInt!
+  "Total depsotis - total withdrawals"
+  currentPrincipal: BigDecimal!
+  "Total rewards added"
+  rewards: BigDecimal!
+  "Total number of rewards added"
+  rewardCount: BigInt!
+  "CurrentPrincipal in USD"
+  TVL: BigDecimal!
+
+  "Day data for tenderFarm"
+  dayData: [TenderFarmDay!]! @derivedFrom(field: "tenderFarm")
+}
+
+type TenderizerDay @entity {
+  "<Timestamp rounded to current day by dividing by 86400>_<portocol>"
+  id: ID!
+  "Tenderizer"
+  tenderizer: Tenderizer!
+  "The date beginning at 12:00am UTC"
+  date: Int!
+
+  "deposits volume in tenderizer for the day"
+  deposits: BigDecimal!
+  "withdraw volume in tenderizer for the day"
+  withdrawals: BigDecimal!
+  "Total volume for the day: deposits - withdrawals"
+  volume: BigDecimal!
+  "cumulative deposit on that day"
+  cumulativeVolume: BigDecimal!
+
+  "staking rewards collected from protocol for the day"
+  rewards: BigDecimal!
+  "cumulative staking rewards collected"
+  cumulativeRewards: BigDecimal!
+
+  "Principle on the first event of the day"
+  startPrinciple: BigDecimal!
+  "Day APY - rewards/principle %"
+  APY: BigDecimal!
+}
+
+type TenderFarmDay @entity {
+  "<Timestamp rounded to current day by dividing by 86400>_<portocol>"
+  id: ID!
+  "TenderFarm"
+  tenderFarm: TenderFarm!
+  "The date beginning at 12:00am UTC"
+  date: Int!
+
+  "farm volume for the day"
+  deposits: BigDecimal!
+  "unfarm volume for the day"
+  withdrawals: BigDecimal!
+  "Total volume for the day: farm - unfarm"
+  volume: BigDecimal!
+  "cumulative farm on that day"
+  cumulativeVolume: BigDecimal! 
+  "rewards added for the day"
+  rewards: BigDecimal!
+  "cumulative rewards on that day"
+  cumulativeRewards: BigDecimal!
+
+  "harvest volume for the day"
+  harvest: BigDecimal!
+  "cumulative harvest on that day"
+  cumulatinveHarvest: BigDecimal!
+
+  "principle in Token (not BPT) value at the start of the day"
+  startPrinciple: BigDecimal!
+  "Day APY - total rewards / total volume %"
+  APY: BigDecimal!
+}
+
+type User @entity {
+  "Wallet address of user"
+  id: ID!
+  "Data for each integration per user"
+  protocols: [UserProtocol!]! @derivedFrom(field: "user")
+}
+
+type UserProtocol @entity { 
+  "<wallet-address>_<portocol>"
+  id: ID!
+  "Protocol"
+  protocol: Protocol!
+  "User"
+  user: User!
+
+  "Total amount depositied in tenderizer by user"
+  tenderizerDeposits: BigDecimal!
+  "Total amount withdrawn from tenderizer by user"
+  tenderizerWithdrawals: BigDecimal!
+
+  "Total amount depositied in farm by user"
+  farmDeposits: BigDecimal!
+  "Total amount withdrawn from farm by  user"
+  farmWithdrawals: BigDecimal!
+  "Total amount harvest collected from farm by  user"
```

Comment:
These data points are useless for userrs and if `farmHarvest` (naming can be better) is the only remaining field that is specific here then that tells us this structure needs redoing.